app/src/main/java/com/adammcneilly/toa/core/ui/components/navigation/TOANavigationDrawerContent.kt

```diff
@@ -0,0 +1,78 @@
+package com.adammcneilly.toa.core.ui.components.navigation
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.sizeIn
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationDrawerItem
+import androidx.compose.material3.PermanentDrawerSheet
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.adammcneilly.toa.ExcludeFromJacocoGeneratedReport
+import com.adammcneilly.toa.core.ui.theme.TOATheme
+
+@Composable
+fun TOANavigationDrawerContent(
+    navigationConfig: TOANavigationConfig,
+    modifier: Modifier = Modifier,
+) {
+    PermanentDrawerSheet(
+        modifier = modifier.sizeIn(
+            minWidth = 200.dp,
+            maxWidth = 200.dp,
```

Comment:
These hardcoded numbers are sketchy, can we do this differently? 

Also may want to consider padding within this sheet in the future so buttons aren't going right to edge of the screen.